shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -100,16 +93,6 @@ private void vibrateHapticFeedback() {
         view.performHapticFeedback(HapticFeedbackConstants.LONG_PRESS);
     }
 
-    private void launchURL(String url) {
-        try {
-            Intent launchIntent = new Intent(Intent.ACTION_VIEW);
-            launchIntent.setData(Uri.parse(url));
```

Comment:
If this is the last remaining use of `Uri`, please remove the import at top of file.